nav2_constrained_smoother/src/constrained_smoother.cpp

```diff
@@ -21,7 +21,7 @@
 #include <vector>
 
 #include "nav2_constrained_smoother/constrained_smoother.hpp"
-#include "nav2_core/exceptions.hpp"
+#include "nav2_core/planner_exceptions.hpp"
```

Comment:
Why not using normal exceptions for the smoothers? Seems odd to use the planner-type exceptions?